### DISCUSSION TITLE
Did CRON_SECRET recently get updated?

### DISCUSSION THREAD

**Discussion by jameshawkinsjr** [vercel/community#6183] [Help] [answered]
### Summary

I have a few cron jobs running for the past few months. I've been using the implementation for "securing" these endpoints with the environmental variable `CRON_SECRET`, per [this documentation](https://vercel.com/docs/cron-jobs/manage-cron-jobs#securing-cron-jobs)

However, today these endpoints started returning "401 Unauthorized". I made no changes to these endpoints or environmental variables in weeks, however it does seem to coincide with a push to production. I've tried redeploying production with no luck.

If I log the headers, I don't see my `CRON_SECRET` in the headers, however I never inspected them to begin with, so I'm not sure what I'm looking for (aside from my CRON_SECRET, which is not there).

My first error log appears at `03/19/2024 17:45 UTC` (2:45pm MST) Did something recently change with the headers for cron jobs? 

### Steps to Reproduce

vercel.json
```
{
    "crons": [
        {
            "path": "/api/path/to/endpoint",
            "schedule": "* 7 * * *"
        },
    ]
}

```

```
export default async (req: NextApiRequest, res: NextApiResponse) => {
    if (req.headers['authorization'] !== `Bearer ${process.env.CRON_SECRET}`) {
        return res.status(401).end('Unauthorized')
    }

    [APPLICATION LOGIC]
    return res.status(200).json({ success: true });

}
```

**Comment by swarnava** [comment]
Hi, we are investigating this and will share more update here: https://www.vercel-status.com/incidents/kg9fnn6hbp9d

**Comment by sebastianroming** [comment, marked as answer]
Hi @jameshawkinsjr,
A fix for this issue has been implemented. You may need to trigger a new deployment to restore the cron job functionality.

**Reply by jameshawkinsjr** [reply]
Thank you!

**Comment by TeoChi-CF** [comment]
I still encounter the same issue
```
export async function GET(request: Request) {
  ...
  console.log(
    request.headers.get("authorization"),
    `Bearer ${process.env.CRON_SECRET}`
  );
...
}
```
I get the cron_secret but no auth header
